### DISCUSSION TITLE
Peacock - window size issue

### DISCUSSION THREAD

**Discussion by xingz711** [idaholab/moose#16097] [Q&A Tools] [answered]
Hello,

Not sure if this is only happening to my laptop. But I couldn't see the entire Peacock interface. Somehow, the buttons at the bottom are not shown completely.  Tried to reduce the window size with the mouse - didn't work.

Any solutions here?

Thanks,
Xing

**Comment by dschwen** [comment]
Yes, the left side panel has a minimum height this is a known issue :-(
It would be great if a scroll bar could be added to it. 🥺 ... @aeslaughter :-)

I took a stab at it, but I'm too unfamiliar with PyQt and, quite honestly I thought we could just sit that problem out, as display resolutions get larger and larger. But it looks like we can't.

**Comment by aeslaughter** [comment, marked as answer]
I am hopeful for some major Peacock improvements this year, but don't know if that will happen for sure.

**Comment by xingz711** [comment]
Ha, thanks for the clarification. Guess I have to work with this for the moment.
